### ISSUE TITLE
September Preview issues

### ISSUE THREAD

**Issue by johndcollins** [Microsoft/AL#664]
I'm having issues with the latest September preview downloaded after the update from Freddy.

The Navserver is not responding to port 45. I think it might be a timeout issue and it's because the Navserver is really slow to respond.

1. I can't download symbols as it times out and the connection closes before it completes.
2. Using the classic dev client, the designer takes up to 10-20 min to come up. 
3. Using the classic dev client I can't compile or delete any objects. I get the following error :
---------------------------
Microsoft Dynamics NAV Development Environment
---------------------------
Unable to process table changes because the request cannot be processed by the specified Microsoft Dynamics NAV Server instance:

Server Name: navserver
Server Instance: nav
Management Port: 7045

Check that:
1. The Microsoft Dynamics NAV Server is running
2. There is only one tenant mounted on the server instance.
---------------------------
OK   
---------------------------

However I checked the Navserver and it's running and management is turned on:

ServerInstance : MicrosoftDynamicsNavServer$NAV
DisplayName    : Microsoft Dynamics NAV Server [NAV]
State          : Running
ServiceAccount : NT AUTHORITY\SYSTEM
Version        : 11.0.18323.0
Default        : True

4. I'm having issues with the export to the new format. The export of objects gets stuck if the objects have a .Net variable. This also happens if I try and export to text from the classic dev client.

**Comment by johndcollins** [comment]
In regards to loading symbols, I tried loading the symbols from an external computer and that worked as expected, so it seems to be an issue on the local computer communicating with the Navserver for all ports.
Maybe a firewall issue?

**Comment by freddydk** [comment]
You cannot use http locally if it is setup for https globally.
Locally, you need to use https - and probably the same url as globally.
Reg. Classic development environment, then the developer preview is not optimized for classic development and there are things, which won't work at this time.
The primary reason for including the classic development environment was to allow people to look at the source, which I understood was necessary.
I will note your scenarios down and have a look whether we can fix those, but else you can download the latest CTP DVD and use that for classic development.

**Comment by johndcollins** [comment]
Ok, I can understand the Classic Client issues, although it would be easier to use this when converting old objects to AL. Right now I have old table objects in my database I was able to import and convert, but now I can't delete them.

The Symbol download is still an issue though. I can't seem download the symbols locally no matter what url combination I use.

**Comment by johndcollins** [comment]
If I use the public url with I get this error:
[2017-09-12 07:28:16.10] The request for path /NAV/dev/packages?publisher=Microsoft&appName=US&versionText=11.0.0.0 failed with code InternalServerError. Reason: The tenant 'default' is not accessible.
[2017-09-12 07:28:16.11] The request for path /NAV/dev/packages?publisher=Microsoft&appName=System&versionText=11.0.0.0 failed with code InternalServerError. Reason: The tenant 'default' is not accessible.
[2017-09-12 07:28:16.11] Could not download reference symbols. Please ensure that:
1. The correct server name and instance are specified in the launch.json file.
2. The correct application locale and version are specified in the app.json file.

**Comment by StanislawStempin** [comment]
Could you please try removing the "tenant" setting from the launch.json completely?

**Comment by johndcollins** [comment]
Same errors without the tenant setting.

**Comment by StanislawStempin** [comment]
Ok, last thing to try - can you check with "tenant" : ""

**Comment by johndcollins** [comment]
That's what it's been all along.

**Comment by johndcollins** [comment]
@Freddy, the dev client is nice to look at code, but if it takes 10 min to open a designer then it's not very useful IMHO.
I hope you can find a way to fix it to it works with the current setup.

**Comment by johndcollins** [comment]
That's the value it's been all along and not working.

**Comment by johndcollins** [comment]
When I try to publish the app from the remote computer which are able to download the symbols, I get this error:
The request for path /NAV/dev/apps failed with code InternalServerError. Reason: The tenant 'default' is not accessible.

**Comment by StanislawStempin** [comment]
I have just deployed a new VM and followed the steps as described on the lading page for September Update VM (Microsoft Dynamics NAV "Tenerife" Developer Preview 11.0.18323.0, VSIX version 0.9.12794)

I was able to successfully download symbols, publish an app and launch the web client both from the VM itself and from a remote machine (although it took 90s for symbols download remotely, that might be a separate issue). 

In both cases I used launch.json as shown on the landing page,  it worked both with tenant set to blank and with tenant parameter removed.

launch.json settings:
"server": "https://xxxxxx.eastus.cloudapp.azure.com",      
"serverInstance": "NAV",      
"tenant": "",      
"authentication": "UserPassword",
app.json settings:
"locale": "US",


I have just noticed that you logged multiple issues under one ticket. Regarding #4 that you logged, can you give an example of a file that gets stuck on ExportToNewSyntax?

**Comment by johndcollins** [comment]
So the reason I put this all in one issue is that I think it's all related.

I think the reason this is happening is because I imported my custom objects into the database to get them converted to AL, but since the system can't communicate with the management port the database not completely in sync. This is why I'm probably getting the tenant error. I can't delete my objects because of the error in the first post.

**Comment by johndcollins** [comment]
In regards to the export to text, what email can I send you the .fob to?

**Comment by StanislawStempin** [comment]
Thanks, John. We received the .fob and looking into it.

**Comment by dkatson** [comment]
I have the same issue. After importing .fob to database i cannot download symbols.

**Comment by dkatson** [comment]
But I had the ability to delete my imported objects (including tables). After this I have downloaded symbols without problems.

Also I noticed one bug.
On my landing page local is FINUS
![image](https://user-images.githubusercontent.com/29101017/30699087-6e54bc7a-9eec-11e7-9356-d80308970aa5.png)

But with this setting I cannot download symbols.
If I change this to 
![image](https://user-images.githubusercontent.com/29101017/30699113-87abbbe2-9eec-11e7-97ea-ffd03055f6c9.png)

Then I can download symbols
